deployments/aws-lambda/src/main/java/org/apache/flink/statefun/playground/aws/GreeterFn.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.statefun.playground.aws;
+
+import java.util.concurrent.CompletableFuture;
+import org.apache.flink.statefun.sdk.java.Context;
+import org.apache.flink.statefun.sdk.java.StatefulFunction;
+import org.apache.flink.statefun.sdk.java.StatefulFunctionSpec;
+import org.apache.flink.statefun.sdk.java.TypeName;
+import org.apache.flink.statefun.sdk.java.message.Message;
+
+public final class GreeterFn implements StatefulFunction {
+
+  static final TypeName TYPENAME = TypeName.typeNameOf("aws.lambda.example", "greeter");
+  static final StatefulFunctionSpec SPEC =
+      StatefulFunctionSpec.builder(TYPENAME).withSupplier(GreeterFn::new).build();
+
+  @Override
+  public CompletableFuture<Void> apply(Context context, Message message) {
+    System.out.println("Welcome, " + message.asUtf8String() + "!");
```

Comment:
Can you replace that with .rawValue converted to UTF8 string?
So that if folks will take it as a baseline, they won't be surprised.

Reply:
Replacement is necessary for the example to work. 
`System.out.println("Welcome, " + message.rawValue() + "!");`

In the messages there are bytes that are not decoded, resulting in an error: _java.lang.IllegalStateException: Not a StringWrapper_.

Reply:
@gabriel2306
This depends on what you are sending to the function.
Keep in mind that `nessage.asUtf8String()` is for StateFun strings (which has its own encoding format).
If you are writing plain utf8 strings, then indeed you would bump into this error.
In this case, you may define your own `Type`, or access it directly as raw bytes as you found out.

Reply:
Yes, I was sending my own types, so for that I was seeing this error.

Thank you so much @tzulitai!